### PR TITLE
Add test cases for policy-role and policy-rolebinding

### PR DESCRIPTION
**Pull request by ycao56**
Resolves https://github.com/stolostron/backlog/issues/20839
Resolves https://github.com/stolostron/backlog/issues/20840

### CODE REVIEW

File: test/integration/policy_rolebinding_test.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+		By("Checking if the status of the root policy is Compliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyRoleBindingName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.Compliant))
+	})
+
+	It("The Role should exist", func() {
+		By("Checking the Role in the " + policyRoleBindingNSName + " namespace")
```
Comment (dhaiducek) [inline test/integration/policy_rolebinding_test.go:106]:
```suggestion
	It("The RoleBinding should exist", func() {
		By("Checking the RoleBinding in the " + policyRoleBindingNSName + " namespace")
```

File: test/integration/policy_role_test.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the policy-role policy", Label("policy-collection", "stable"), func() {
+	const (
+		policyRoleName   = "policy-role"
+		policyRoleURL    = policyCollectACURL + policyRoleName + ".yaml"
+		policyRoleNSName = policyRoleName + "ns"
+		roleName         = "sample-role"
+	)
```
Comment (dhaiducek) [inline test/integration/policy_role_test.go:26]: This is good. I was thinking the other test files shouldn't have package-level variables for their names.

Comment (ycao56) [reply test/integration/policy_role_test.go:26]: yep

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**